sharding/nodesCoordinator/indexHashedNodesCoordinator.go

```diff
@@ -243,6 +232,14 @@ func checkArguments(args ArgNodesCoordinator) error {
 		return ErrNilEpochStartStaticStorer
 	}
 
+	err := core.CheckHandlerCompatibility(args.EnableEpochsHandler, []core.EnableEpochFlag{
```

Comment:
I would keep this right after `if check.IfNil(args.EnableEpochsHandler) {`

Reply:
changed